common/models/zod-schemas.ts

```diff
@@ -41,3 +41,17 @@ export const OttApiRequestAddToQueueSchema = z.union([
 export const OttApiRequestRemoveFromQueueSchema = z.object({
 	...VideoIdSchema.shape,
 });
+
+export const OttApiRequestAccountRecoveryStartSchema = z.union([
+	z.object({
+		email: z.string().email(),
```

Comment:
if we change this validation to `.email().min(3)` can we avoid the unit test changes? Ideally those shouldn't need to change because 3 character email addresses like `a@b` _are_ valid emails, and it's not like we require email validation on signup.

Reply:
We need to tack on `.com` or something similar, but other than that, yes.
See [here](https://colinhacks.com/essays/reasonable-email-regex).

Reply:
Alright, I'm fine with this then.